src/main/java/de/doubleslash/keeptime/common/Resources.java

```diff
@@ -65,7 +65,9 @@ public enum RESOURCE {
 

```

Comment:
can you delete the old FXML_ABOUT constant here and with it also the empty about.fxml file?